force-app/test/classes/MethodSpyTest.cls

```diff
@@ -108,7 +108,7 @@ private class MethodSpyTest {
   static void givenSpyConfiguredWithAMixOfObjectAndParamMatcher_whenCalledWithAllMatching_spyIsCalled() {
```

Comment:
**question:** do we still have test when passing `Params` ?

Reply:
No we don't, I can add some to be sure that this is covered